### PR TITLE
Update basket functionality

### PR DESCRIPTION
**Pull request by nustiueudinastea**
Implemented functionality for updating the item quantity in the basket.

### CODE REVIEW

File: api/cart/index.js [hunk 1]
```diff
@@ -61,6 +61,32 @@
...
+    if (req.params.id == null) {
+      return next(new Error("Must pass id of item to update"), 400);
+    }
+
+     console.log("Updating item: " + req.params.id + " quantity: " + req.body.quantity);
+
+    var custId = helpers.getCustomerId(req, app.get("env"));
+    var options = {
+      uri: endpoints.cartsUrl + "/" + custId + "/items",
+      method: 'PATCH',
```
Comment (vlal) [inline api/cart/index.js]: Shouldn't this be a PUT?

Comment (nustiueudinastea) [inline api/cart/index.js]: It's implemented as a PATCH in the cart service, and that makes sense actually. PATCH is used usually when updating an item.

Comment (vlal) [inline api/cart/index.js]: my bad you're right

**Comment by nustiueudinastea** [comment]
Implements https://github.com/microservices-demo/microservices-demo/issues/36

**Pull request merged**